src/ocean/core/SmartUnion.d

```diff
@@ -226,6 +253,34 @@ private struct SmartUnionIntern ( U )
      **************************************************************************/
 
     Active active;
+
+    /***************************************************************************
+
+        List of active state names
+
+    ***************************************************************************/
+
+    const istring[] active_names = member_string_list();
+
+    /***************************************************************************
+
+        CTFE function to generate the list of active state names for union U.
+
+        Returns:
+            a list containing the names of each of the active states of the
+            smart-union (i.e. the names of the fields of U)
+
+    ***************************************************************************/
+
+    static private istring[] member_string_list ( )
+    {
+        istring[] names = ["none"[]];
```

Comment:
The smart-union enum member is actually called `none`, so this is entirely accurate. https://github.com/gavin-norman-sociomantic/ocean/blob/52961089d40181042c1680ad7af43166c4d34861/src/ocean/core/SmartUnion.d#L247